docs/userguide/running-unit-tests.md

```diff
@@ -325,6 +325,8 @@ Multiple tags are separated by comma.
 
 Tag expressions are boolean expressions created by combining tags with the `!`, `&`, `|` operators. Tag expressions can be grouped using `(` and `)` braces. Grouping tag expressions affects operator precedence.
 
+Two special expressions are supported, `any` and `none`, which select all tests with any tags at all, and all tests without any tags, respectively. These special expressions may be combined with other expressions just like normal tags. When using `none` be aware that if the suite is tagged it will exclude any tests and children belonging to that suite.
```

Comment:
Two reserved keywords, `any` and `none`, can be used when creating a tag expression to run tests. 
- `any` keyword represents tests and suites with any tags 
- `none` keyword represents tests and suites without tags

These keywords may be combined with regular tags.

!!! note
    When specifying `none`, be aware that it will exclude any tests/suites/contexts contained within a tagged suite.

Reply:
The mkdocs that we use to generate html documentation will interpret the below lines as a special "note" block 
```
!!! note
    When specifying `none`, be aware that it will exclude any tests/suites/contexts contained within a tagged suite.
```

We will get a nice, more visible Note indicators
![image](https://user-images.githubusercontent.com/1398171/234957632-fc461e32-3de7-4f50-86e9-31cfd736685e.png)

Reply:
Ahh didnt realize. updated.